### PR TITLE
Try reading version from .ruby-version if not supplied

### PR DESCRIPTION
**Pull request by vassilevsky**

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -168,7 +169,11 @@ func Init() {
 			installCommand(args[0])
 		}
 	} else {
-		listCommand()
+		if fsutil.CheckPerms("FRS", VERSION_FILE) {
```
Comment (andyone) [inline cli/cli.go]: How I can list versions with dir which contains version file?

Comment (vassilevsky) [inline cli/cli.go]: I was sure there was a `list` command that was run by default 😕 The `listCommand` function name kinda suggests it.

Should I add the `--list, -l` switch that runs `listCommand`?

Comment (andyone) [inline cli/cli.go]: Look at my implementation in #11

**Comment by andyone** [comment]
This feature implemented in #11 Thanks for nice idea.

**Pull request closed without merging**